cozy/cost_model.py

```diff
@@ -34,11 +34,12 @@ def order_objects(x, y):
 
 class CostModel(object):
 
-    def __init__(self, assumptions : Exp = T, examples=(), funcs=()):
+    def __init__(self, assumptions : Exp = T, examples=(), funcs=(), freebies : [EStateVar] = []):
```

Comment:
Here (and elsewhere): `freebies` isn't technically made of `EStateVar`s. Each one is a "state expression"---the `.e` property of an `EStateVar`. There isn't a proper type for state expressions; using `Exp` instead of `EStateVar` is fine.